sdk/eventgrid/azure-resourcemanager-eventgrid/src/main/samples/java/README.md

```diff
@@ -0,0 +1,74 @@
+---
+page_type: sample
+languages:
+  - java
+products:
+  - azure
+  - azure-event-grid
+urlFragment: eventgrid-resourcemanager-samples
+---
+
+# Azure Event Grid Publish/Consume Samples for Java
+
+This contains Java samples for publishing events to Azure Event Grid and consuming events from Azure Event Grid. 
+It also contains a set of management samples that demonstrates how to manage topics and event subscriptions using Java code.
+
+## Features
+
+These samples demonstrate the following features:
+
+- How to create an event grid topic using Java
+- How to create an event subscription to a topic using Java.
+- How to create an event hub using Java.
+- How to publish events to Azure Event Grid using Java.
+- How to consume events delivered by Azure Event Grid through an Azure Event Hub consumer using Java.
+
+## Getting started
+
+Getting started explained in detail [here][EVENTGRID_README_GETTING_STARTED]. 
+Please refer it to add dependency and configure authentication environment variables.
+
+For details on including this dependency in other build tools (Gradle, SBT, etc), refer [here](https://search.maven.org/artifact/com.azure/azure-core).
+
+## Sample Details
+
+Azure event grid publish and consume sample is in 'sdk/eventgrid/azure-resourcemanager-eventgrid/src/main/samples/java/com/azure/resourcemanager/eventgrid/EventGridPublishAndConsumeExample.java',
+it has the `main()` function and can be run directly.
+
+The sample demonstrate following operations:
+
+1. Create a resource group.
+2. Create an event hub.
+3. Create an event grid topic.
+4. Create an event grid subscription.
+5. Retrieve the event grid client connection key.
+6. Create an event grid publisher client: `EventGridPublisherClient`.
+7. Create an event hub consumer client: `EventHubConsumerAsyncClient`. 
+8. Subscribe to coming events from event grid using `EventHubConsumerAsyncClient`.
+9. Publish custom events to the event grid using `EventGridPublisherClient`.
+10. Clean up the resources created above.
+
+## Key concepts
+
+Key concepts are explained in detail [here][EVENTGRID_README_KEY_CONCEPTS].
```

Comment:
You can add link to `detail` and remove `here`.

Reply:
This is for consistency purpose, readme for other samples keep the format like: Key concepts are explained in detail [here][EVENTGRID_README_KEY_CONCEPTS].